src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -748,7 +748,7 @@ override public object ExecuteScalar()
             {
                 statistics = SqlStatistics.StartTimer(Statistics);
                 SqlDataReader ds;
-                ds = RunExecuteReader(0, RunBehavior.ReturnImmediately, true, ADP.ExecuteScalar);
+                ds = RunExecuteReader(0, RunBehavior.ReturnImmediately, true, nameof(ExecuteScalar));
```

Comment:
Wouldn't it make more sense to change RunExecuteReader to take [CallerMemberName] string method=""
You can get rid of most of the nameof(...) being passed down to RunExecuteReader as well.